examples/crm/src/contacts/ContactAside.tsx

```diff
@@ -167,6 +170,7 @@ export const ContactAside = ({ link = 'edit' }: { link?: 'edit' | 'show' }) => {
                 </ReferenceManyField>
                 <AddTask />
             </Box>
+            <DeleteButton redirect={location.state?.from || undefined} />
```

Comment:
**suggestion:** Use a `DeleteWithConfirmButton` instead to confirm debore deletion

Reply:
The undoable is used everywhere else, so for me it's important to remain consistent.